store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10054,4 +10054,9 @@ TODO: delete them permanently from here
   <desc>Help URL for modern client</desc>
 </attr>
 
+  <attr id="4024" name="zimbraBlockEmailSendFromImapPop" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountInfo,accountCosDomainInherited" since="11.0.0">
```

Comment:
Please change ID. 4024 and 4025 will be added on https://github.com/Zimbra/zm-mailbox/pull/1426

Reply:
Done.

Reply:
thank you for the update.